feat(curriculum): dice game project (#50819)
freeCodeCamp/freeCodeCamp: curriculum/challenges/english/15-javascript-algorithms-and-data-structures-22/learn-intermediate-algorithmic-thinking-by-building-a-dice-game/64a1b6a63614f002dce978ed.md

---
id: 64a1b6a63614f002dce978ed
title: Step 26
challengeType: 0
dashedName: step-26
---

# --description--

> Create and `else` statement that adds the `textContent` property on `rulesBtn` to display `Show Rules`.

Reviewer: ```suggestion
Create an `else` statement that assigns the `textContent` property on `rulesBtn` the value  `Show Rules`.
```

# --hints--

> You should add the `textContent` property on `rulesBtn` to display `Show Rules` inside and `else` statement.

Reviewer: ```suggestion
You should update the `textContent` property on `rulesBtn` to the string `Show Rules` inside an `else` statement.
```

```js

> assert.match(code, /else\s*{\s*.*?rulesBtn\.textContent\s*=\s*['"]Show Rules['"]/s);

Reviewer: ```suggestion
assert.match(code, /else\s*{\s*.*?rulesBtn\.textContent\s*=\s*('|"|`)Show Rules\1/s);
```
